app/src/main/res/layout/activity_create_profile.xml

```diff
@@ -9,21 +10,14 @@
         android:layout_height="wrap_content"
         android:orientation="vertical">
 
-        <android.support.v7.widget.Toolbar
-            android:id="@+id/toolbar3"
+        <ImageView
+            android:id="@+id/large_logo"
             android:layout_width="match_parent"
-            android:layout_height="?attr/actionBarSize"
-            android:background="@android:color/holo_blue_dark">
-
-            <TextView
-                android:id="@+id/toolbar_title"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:layout_gravity="center"
-                android:text="@string/app_name"
-                android:textSize="@dimen/toolbar_title_size"
-                android:textStyle="bold" />
-        </android.support.v7.widget.Toolbar>
+            android:layout_height="match_parent"
+            android:layout_gravity="center"
+            android:contentDescription="@string/logo"
+            app:srcCompat="@drawable/large_logo_with_shadow"
```

Comment:
app:srcCompat is used for vector drawables like SVG's. For all other images you should use android:src